instr.ml

```diff
@@ -398,12 +404,13 @@ class map = object (m)
       Assert (m#expression e)
     | Stop e ->
       Stop (m#expression e)
-    | Osr {label; cond; target; map} ->
+    | Osr {label; cond; target; var_map; frame_maps} ->
       Osr {
         label;
```

Comment:
Why do we need it *now* would be my guideline. It would be dead code at the moment since we do not use this flexibility. My software engineering practice and also training has thought me to never ever add features that might come handy in the future. It's just half-dead code that slows down refactorings. Like in this case I had to rewrite the existing code that was too flexible for it's own good. If I spent more than 2' trying to figure out how to abstract it even though I am sure we won't use it in the next 3 weeks that would be a waste. But what's worse is I add a badly tested (since unused) feature to the codebase. The more things in the codebase the harder it is to move it. I know that is now a completely overblown answer to this small point. But I thought it's a good situation to explain myself and why I sometimes remove things that I see as cruft.